icicle-compiler/src/Icicle/Sea/FromAvalanche/Base.hs

```diff
@@ -91,7 +91,7 @@ mangleToSeaName = SeaName . Text.pack . concatMap mangle . show . pretty
      | c >= 'A' && c <= 'Z'
      = [c]
      | c == '_'
-     = "__"
+     = "_"
```

Comment:
Should we have to pick one, between:

`forall x. mangle (mangle x) == mangle x`

and:

`mangle x == mangle y ==> x == y`

?